.github/workflows/cypress_e2e.yml

```diff
@@ -0,0 +1,35 @@
+name: Cypress E2E Tests
+
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+
+env:
+  CI: true
```

Comment:
FYI I think this is already set by default: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables